poly-benches/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
```

Comment:
After this PR was opened, PR #558 was created and merged. That PR deletes the `poly-benches` crate, so this file shouldn't exist. Please delete this file and the enclosing folder =)

Reply:
Got it! should be removed now. Must have gotten pulled along in merging or something.